packages/uniroll/src/plugins/omt.ts

```diff
@@ -11,17 +11,18 @@
  * limitations under the License.
  */
 
-const MagicString = require("magic-string");
+import { PluginImpl } from "rollup";
```

Comment:
I've converted this js to ts for passing tests.
https://github.com/mizchi/uniroll/pull/10/checks?check_run_id=906145631